baracoda/orm/barcodes_group.py

```diff
@@ -0,0 +1,27 @@
+from sqlalchemy import Column, Integer, String, DateTime, Sequence
+from sqlalchemy.orm import relationship
+
+from baracoda.orm.base import Base
+
+
+class BarcodesGroup(Base):
+    __tablename__ = "barcodes_groups"
+
+    id = Column(Integer, Sequence("barcodes_groups_id_seq"), primary_key=True)
+    created_at = Column(DateTime)
+    barcodes = relationship("Barcode", back_populates="barcodes_group", uselist=True)  # type: ignore
+
+    def __repr__(self):
+        return "<BarcodesGroup(id='%s', barcodesCount='%s', created_at='%s')>" % (
```

Comment:
`barcodesCount` is camelCase but `created_at`is snake_case?

Reply:
ok, snake_case so is more python style